server/routes/DogRouter.js

```diff
@@ -0,0 +1,10 @@
+const DogRouter = require('express').Router();
+const controller = require('../controllers').dog;
+
+DogRouter.post('/', controller.registerDog);
+DogRouter.get('/:_id', controller.getDogs);
```

Comment:
confused about get dogs (:_id) portion of the route, since we won't be passing id to getAll dogs

Reply:
line 5 as it is would suit the `Profile` page I think, but an additional route would be needed to get many/all dogs